webapp/templates/rewriterule-create.xhtml

```diff
@@ -0,0 +1,176 @@
+                    the problem of transitory URIs in location-based URI schemes like HTTP.</p>
+                    <p>Partial PURLs are a special class of PURL that must end with a '/' and 
+                    allow a hierarchy of URLs to be mapped to another resource. The request pattern
+                    may include regular expressions in the path subsequent to the PURL name.</p>
+                    <p>Other URLs cannot end with a '/' and the request pattern only applies to the query string.</p>
+                </aside>
+            </div>
+        </div>        
+                
+        <div id="ruleOptions" style="display:none">
+            <!-- These are not used by the template but are required for validation when the resource is created -->
+            <input type="hidden" property="calli:copy" content="?copy" value="{?copy}"/>
+            <input type="hidden" property="calli:canonical" content="?canonical" value="{?canonical}"/>
+            <input type="hidden" property="calli:alternate" content="?alternate" value="{?alternate}"/>
+            <input type="hidden" property="calli:describedby" content="?describedby" value="{?describedby}"/>
+            <input type="hidden" property="calli:resides" content="?resides" value="{?resides}"/>
+            <input type="hidden" property="calli:moved" content="?moved" value="{?moved}"/>
+            <input type="hidden" property="calli:missing" content="?missing" value="{?missing}"/>
+            <input type="hidden" property="calli:gone" content="?gone" value="{?gone}"/>
+            <input type="hidden" property="calli:illegal" content="?illegal" value="{?illegal}"/>
+            <input type="hidden" property="calli:post" content="?post" value="{?post}"/>
+            <input type="hidden" property="calli:put" content="?put" value="{?put}"/>
+            <input type="hidden" property="calli:patch" content="?patch" value="{?patch}"/>
+            <input type="hidden" property="calli:delete" content="?delete" value="{?delete}"/>            
+        </div>
+                
+        <div class="row">
+            <div class="col-sm-12">
+                <label>Rules
+                    <a href="javascript:void(0)" title="More" onclick="addRule('#rules')" class="glyphicon glyphicon-plus" />
+                </label>
+                <table id="ruleTable" class="table-condensed col-sm-12">
+                    <thead>
+                        <tr>
+                            <th class="col-sm-5" colspan="2">Request</th>
+                            <th class="col-sm-6" colspan="2">Response</th>
+                            <th class="col-sm-1"></th>
+                        </tr>
+                        <tr>
+                            <th class="col-sm-2">Methods</th>
+                            <th class="col-sm-3">Pattern</th>
+                            <th class="col-sm-2">Method/Status</th>
```

Comment:
The think the term "Action" is a better fit here to indicate that the system will be initiating sub-requests (not the client)